ffmpeg/loopback.go

```diff
@@ -40,8 +45,14 @@ func (l *loopback) Start() error {
 		if err := cmd.Start(); err != nil {
 			return err
 		}
-		// FIXME: Starting ffmpeg takes some time
-		time.Sleep(1 * time.Second)
+
+		// Starting ffmpeg takes some time
+		for !strings.Contains(l.output.String(), "frame=") {
```

Comment:
Does `l.output.String()` waits until there are bytes in the buffer? 
The docs for [func (Bytes) String](https://golang.org/pkg/bytes/#Buffer.String) actually doesn't tell.

Reply:
Where you able to test this?
I can't get the Camera to start broadcasting, and haven't figured why.
I also think it's because ffmpeg takes some time.

Reply:
It does indeed take some time to start the broadcast. I am running the patched version for a few months now mostly without problems.
However, at the moment I am not near the camera and don't want to touch the working setup. Hence, I have not been able to suggest a proper patch which incorporates the feedback above. I am going to look into that in a few weeks and update this PR.

Reply:
@tkintscher Any news on this issue?